rpm/s2n-tls.spec

```diff
@@ -0,0 +1,60 @@
+# SPEC file overview:
+# https://docs.fedoraproject.org/en-US/quick-docs/creating-rpm-packages/#con_rpm-spec-file-overview
+# Fedora packaging guidelines:
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/
+
+Name:           s2n-tls
+Version:        1.0.18
+Release:        0%{?dist}
+Summary:        A C99 TLS library
+
+License:        Apache2.0
+URL:            https://github.com/aws/%{name}
+%undefine       _disable_source_fetch
+Source0:        https://github.com/aws/%{name}/archive/v%{version}.tar.gz
+%define         SHA256SUM0 e548c1208205a8959c68f8389eac721fc37fc99d3600790430783972272c9452
+Patch0:         makefile.patch
+Group:          System Environment/Libraries
+BuildRequires:  openssl11-static cmake3 ninja-build zlib-devel
+%description
+A C99 TLS library
+
+%package libs2n
+Summary: A C99 compatable TLS library
+%description libs2n
+%{name} is a C99 implementation of the TLS/SSL protocols that is designed to be simple, small, fast, and with security as a priority.
+
+%package devel
+Summary: Header files for %{name}
+%description devel
+%{name} is a C99 implementation of the TLS/SSL protocols that is designed to be simple, small, fast, and with security as a priority.
+
+%prep libs2n
+%setup -q
+# TODO: remove the patch after the release of this change.
+# We need this patch b/c we're pinning the source to a fixed github release, Makefile needs updating for rpmbuild to work.
+%patch0 -p1
+
+%build libs2n
+cmake3 -GNinja -DS2N_INTERN_LIBCRYPTO=on \
```

Comment:
Where is this RPM going to be used? I think distros generally frown upon static linking or interning because it's harder to patch when there is an update.

Reply:
yes, reverted.